kpatch-build/kpatch-build

```diff
@@ -221,25 +221,25 @@ find_special_section_data_ppc64le() {
 		       e == 1 {printf("export EX_STRUCT_SIZE=%d\n", $4); e = 2}
 
 		       # Bail out once we have everything
-		       f == 2 && b == 2 && e == 2 {exit}')
+		       f == 2 && b == 2 && e == 2 {exit}')"
 
-	[[ -n $SPECIAL_VARS ]] && eval "$SPECIAL_VARS"
+	[[ -n "$SPECIAL_VARS" ]] && eval "$SPECIAL_VARS"
 
-	[[ -z $FIXUP_STRUCT_SIZE ]] && die "can't find special struct fixup_entry size"
-	[[ -z $BUG_STRUCT_SIZE ]] && die "can't find special struct bug_entry size"
-	[[ -z $EX_STRUCT_SIZE ]]  && die "can't find special struct exception_table_entry size"
+	[[ -z "$FIXUP_STRUCT_SIZE" ]] && die "can't find special struct fixup_entry size"
+	[[ -z "$BUG_STRUCT_SIZE" ]] && die "can't find special struct bug_entry size"
+	[[ -z "$EX_STRUCT_SIZE" ]]  && die "can't find special struct exception_table_entry size"
```

Comment:
I think all the quotes above (and below) for `SPECIAL_VARS` and `*_STRUCT_SIZE` aren't needed.

Reply:
Dito.